### PR TITLE
feat: add download link button for webui desktop

### LINKED ISSUE #1486: Let's embed direct download link for WebUI desktop in WebUI
### Main idea

Backend.AI GUI is not only limited to the web browser but also has a Desktop application implemented in electron. Sometimes users want to download Desktop App, and every time they ask about it we need to copy and paste the link, which is cumbersome.
Therefore, it's better to add a download button in somewhere in WebUI, so that we can guide it to the users easily. Also, we need to provide the version to download to be the same version of WebUI’s.

### Alternative ideas

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by Sujin-Kim1**
- this resolves #1486 
- Add `appDownloadUrl` option to config.
- Add a button that links to `<appDownloadUrl>/v<webui version>/backend.ai-desktop-<webui-version>-<OS>-<architecture>.<extension>`

image
image
image
image

related PR: https://github.com/lablup/backend.ai/pull/861

### CODE REVIEW

File: src/components/backend-ai-webui.ts [hunk 1]
```diff
@@ -1152,6 +1158,7 @@ export default class BackendAIWebUI extends connect(store)(LitElement) {
               </address>
               <div id="sidebar-navbar-footer" class="vertical start end-justified layout" style="margin-left:16px;">
                 <backend-ai-help-button active style="margin-left:4px;"></backend-ai-help-button>
+                <mwc-icon-button class="download-link-menu-icon fg white" icon="download" slot="graphic" style="margin-left:4px;" @click="${() => this.showDownloadPage()}"></mwc-icon-button>
```
Comment (inureyes) [inline src/components/backend-ai-webui.ts]: It should be only shown on Web mode, not on App mode.

File: src/components/backend-ai-webui.ts [hunk 2]
```diff
@@ -1176,6 +1183,7 @@ export default class BackendAIWebUI extends connect(store)(LitElement) {
             </address>
             <div id="sidebar-navbar-footer" class="vertical start end-justified layout" style="margin-left:16px;">
               <backend-ai-help-button active style="margin-left:4px;"></backend-ai-help-button>
+              <mwc-icon-button class="download-link-menu-icon fg white" icon="download" slot="graphic" style="margin-left:4px;" @click="${() => this.showDownloadPage()}"></mwc-icon-button>
```
Comment (inureyes) [inline src/components/backend-ai-webui.ts]: Here, too.

**Review by inureyes — CHANGES_REQUESTED** [review]
The download button should be only shown on Web mode, not on App mode.

And, how about moving it to another location? e.g. Panel on summary page?

File: src/components/backend-ai-webui.ts (hunk 1 above)
Comment (agatha197) [reply src/components/backend-ai-webui.ts]: reflected!

File: src/components/backend-ai-webui.ts (hunk 2 above)
Comment (agatha197) [reply src/components/backend-ai-webui.ts]: reflected.

**Comment by inureyes** [comment]
@Sujin-Kim1 How about 'Download App' instead of 'Download' ?

**Review by inureyes — APPROVED** [review]
LGTM

**Pull request merged**